core/ops.rs

```diff
@@ -239,6 +242,8 @@ pub struct OpState {
   pub feature_checker: Arc<FeatureChecker>,
   pub external_ops_tracker: ExternalOpsTracker,
   pub op_stack_trace_callback: Option<OpStackTraceCallback>,
+  pub(crate) unrefed_ops: UnrefedOps,
+  pub unrefed_resources: HashSet<ResourceId>,
```

Comment:
should this be exposed instead as something like `uv_unref()` ?

https://github.com/nodejs/node/blob/main/src/handle_wrap.cc#L54